mungegithub/misc-mungers/deployment/kubernetes/configmap.yaml

```diff
@@ -16,7 +16,8 @@ github-key-file: /etc/hook-secret-volume/secret
 # !!! This *MUST* be kept in sync with
 #     submit-queue/deployment/kubernetes/configmap.yaml !!!
 required-retest-contexts: "\
-  pull-kubernetes-bazel,\
+  pull-kubernetes-bazel-build,\
```

Comment:
did you update the SQ?

Reply:
hm, seems like `mungegithub/submit-queue/deployment/kubernetes/configmap.yaml` wasn't updated?

Reply:
oh oh oh oh oh - too many places

and does sq picks up config change automatically now or we need to redeploy that

Reply:
I imagine redeploy is needed, but I'm not sure.

Reply:
config has been updated, I asked @cjwagner about redeploying.